gulpfile.babel.js

```diff
@@ -247,6 +249,19 @@ gulp.task('json', function() {
 });
 
 
+const accessAsync = promisify(fs.access);
+
+
+gulp.task('keycheck', () =>
+  accessAsync('./service-account-key.json')
+  .catch(error => Promise.reject(error +
```

Comment:
Please indent 4 spaces or keep on the previous line.

Reply:
Sorry– should the catch be indented, or the wrapped string? Or both?

Reply:
Fixed